### PR TITLE
Refactor part 2

### PR DESCRIPTION
**Pull request by shine2lay**
- use of beforeEach() in tests for better readability
- addressed @rmerom's outstanding comment from refactor part 1

---
This change is [Reviewable](https://reviewable.io/reviews/wetrustplatform/rosca-contracts/155)

### CODE REVIEW

**Comment by blorange** [comment]
Review status: 0 of 12 files reviewed at latest revision, 2 unresolved discussions.

---

*[test/contributeUnitTest.js, line 22 at r1](https://reviewable.io:443/reviews/wetrustplatform/rosca-contracts/155#-KjFrvnukHDxJsdBdsoA:-KjFrvnukHDxJsdBdsoB:b-bl435r) ([raw file](https://github.com/wetrustplatform/rosca-contracts/blob/d62ddb51da144b8ef3e773bde2653cba37c778c5/test/contributeUnitTest.js#L22)):*
> ```JavaScript
>     beforeEach(co(function* () {
>       roscas = yield utils.createETHandERC20Roscas(accounts);
>       rosca = yield utils.createEthROSCA();
> ```

Do we want to create both of these each time? There are a few lines down below which are redefining `rosca` inside loops

---

*[test/utils/consts.js, line 18 at r1](https://reviewable.io:443/reviews/wetrustplatform/rosca-contracts/155#-KjFsTJQf7AmqFb8Bi3d:-KjFsTJQf7AmqFb8Bi3e:bhz38vw) ([raw file](https://github.com/wetrustplatform/rosca-contracts/blob/d62ddb51da144b8ef3e773bde2653cba37c778c5/test/utils/consts.js#L18)):*
> ```JavaScript
>     memberList = [accounts[1], accounts[2], accounts[3]]
>   },
>   MEMBER_COUNT: function() {
> ```

If we camelCased `memberList`, should we also do this one and `DEFAULT_POT()`

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/wetrustplatform/rosca-contracts/155)*

**Comment by shine2lay** [comment]
Review status: 0 of 12 files reviewed at latest revision, 2 unresolved discussions.

---

*[test/contributeUnitTest.js, line 22 at r1](https://reviewable.io:443/reviews/wetrustplatform/rosca-contracts/155#-KjFrvnukHDxJsdBdsoA:-KjFtY6f5DjgCneUWTY_:bfbp2rz) ([raw file](https://github.com/wetrustplatform/rosca-contracts/blob/d62ddb51da144b8ef3e773bde2653cba37c778c5/test/contributeUnitTest.js#L22)):*
<details><summary><i>Previously, blorange (Brian Loranger) wrote…</i></summary><blockquote>

Do we want to create both of these each time? There are a few lines down below which are redefining `rosca` inside loops
</blockquote></details>

those have different parameter, that's why i didn't delete them. 
re creating : Actually, i think its better to create both each time because we want a clean copy for each test and it helps ensure that. 

---

*[test/utils/consts.js, line 18 at r1](https://reviewable.io:443/reviews/wetrustplatform/rosca-contracts/155#-KjFsTJQf7AmqFb8Bi3d:-KjFyN7dG3qVn-RTqOaT:b-896fix) ([raw file](https://github.com/wetrustplatform/rosca-contracts/blob/d62ddb51da144b8ef3e773bde2653cba37c778c5/test/utils/consts.js#L18)):*
<details><summary><i>Previously, blorange (Brian Loranger) wrote…</i></summary><blockquote>

If we camelCased `memberList`, should we also do this one and `DEFAULT_POT()`
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/wetrustplatform/rosca-contracts/155)*

**Comment by shine2lay** [comment]
@blorange PTAL, i would like to merge this after you approve 

edit: sorry, i didn't realize its already weekend for you, no rush,

**Review by blorange — APPROVED** [review]

**Pull request merged**
